docker/generator.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Change the file name to generate-parachain-specs.sh and move it to the scripts folder